kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/framework/concurrency/concurrency.kt

```diff
@@ -0,0 +1,20 @@
+package io.kotest.framework.concurrency
+
+import io.kotest.common.ExperimentalKotest
+
+typealias Millis = Long // TODO: inline class this in 4.6.x (when we have kotlin 1.5)
```

Comment:
Should we use inline classes? If we rely on 1.5 features we're forcing users to upgrade.

Reply:
You're right 👍 I'm just used to being tied to kotest's kotlin version cause of duration, the whole point of this PR! 😂😅